warehouse/query-core/src/test/java/datawave/query/transformer/GroupingTest.java

```diff
@@ -108,6 +107,7 @@ public static class ShardRange extends GroupingTest {
         protected BaseQueryResponse runTestQueryWithGrouping(Map<String,Integer> expected, String querystr, Date startDate, Date endDate,
                         Map<String,String> extraParms, RebuildingScannerTestHelper.TEARDOWN teardown, RebuildingScannerTestHelper.INTERRUPT interrupt)
                         throws Exception {
+            System.out.println("runTestQueryWithGrouping teardown/interrupt: " + teardown.name() + "/" + interrupt.name());
```

Comment:
Would it be beneficial to use the logger instead for this? Not a major deal either way.

Reply:
+1, in favor of using the logger instead of system.out

Reply:
changed